js/pull-requests.js

```diff
@@ -0,0 +1,38 @@
+var getUserId = document.getElementById("usrid");
+getUserId.addEventListener("keyup", function(event) {
+  event.preventDefault();
+  const con = event.target.value.toLowerCase();
+  var url = "https://api.github.com/repos/rarmatei/js-exercises/pulls";
+  fetch(url)
+    .then(resp => resp.json())
+    .then(function(result) {
+      document.getElementById("pull-requests-list").innerText = "";
+      result.forEach(function(element) {
```

Comment:
I'd avoid using the word `element` as a generic name for array items. It works OK in other contexts, but when you are dealing with HTML/DOM it is likely to cause confusion with [HTML elements](https://www.w3schools.com/html/html_elements.asp). So it confuses me when I see a DOM manipulation function that takes an `element` parameter but the parameter isn't actually a DOM element (as returned by something like `document.getElementById` or `document.createElement`).